packages/shared/src/write-archive/index.ts

```diff
@@ -56,12 +56,23 @@ export async function writeTestResult(
 
       const archiveFile = new ArchiveFile(url, response, pageUrl);
       const origSrcPath = archiveFile.originalSrc();
-      const fileSystemPath = archiveFile.toFileSystemPath();
+      let fileSystemPath = archiveFile.toFileSystemPath();
 
       if (origSrcPath !== fileSystemPath) {
         sourceMap.set(origSrcPath, fileSystemPath);
       }
 
+      // detect the /@fs/ prefix and convert to a relative path that we can add to the archive dir
+      if (fileSystemPath.startsWith('/@fs/')) {
+        const fsPathSlug = fileSystemPath.split('@fs/')[1];
+
+        const overlap = findPathOverlap(fsPathSlug, archiveDir);
+        // with an overlap over a certain length combined with the @fs finding, we can assume a repeated system path
+        if (overlap.length > 30) {
+          fileSystemPath = fsPathSlug.split(overlap)[1];
+        }
```

Comment:
(Opinionated Question, not necessary for merging this) I'm wondering if we need to meet the `30` count for changing the file system path -- would it hurt if that length were much lower? Seems it's possible that the distance from the root of the drive could be much less. But I might be overthinking this.